discretize/unstructured.py

```diff
@@ -0,0 +1,978 @@
+import numpy as np
```

Comment:
wondering on the naming convention for the file. should it be `unstructured_mesh` (e.g. consistent with `tree_mesh`)?